fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/rollcall/CreateRollCall.java

```diff
@@ -127,8 +127,8 @@ public int hashCode() {
         getId(),
         getName(),
         getCreation(),
-        getStartTime(),
-        getStartType(),
+        getProposedStart(),
```

Comment:
If I understand correctly, you removed the StartType enum, to avoid using NOW and SCHEDULED, and directly let the organizer set a start time which would categorize the roll call either in the present section or future section, depending on the time, right ?

Reply:
Exactly